src/compile.js

```diff
@@ -0,0 +1,196 @@
+const clone = require('clone');
+
+const detectTransactionExampleNumbers = require('./detect-transaction-example-numbers');
+const compileUri = require('./compile-uri');
+const getTransactionName = require('./transaction-name');
+const getTransactionPath = require('./transaction-path');
```

Comment:
The move towards `import` is scheduled as a next step?

Reply:
Correct.